### PR TITLE
Demotes HANDSLOW to secondary flag

### PR DESCRIPTION
**Pull request by coiax**
It's used in literally four places, it doesn't need a high speed
bitflag slot.

### CODE REVIEW

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 1]
```diff
@@ -348,27 +348,30 @@
...
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 
-/obj/item/weapon/nullrod/pitchfork
-	icon_state = "pitchfork0"
-	name = "unholy pitchfork"
-	w_class = WEIGHT_CLASS_NORMAL
-	desc = "Holding this makes you look absolutely devilish."
-	attack_verb = list("poked", "impaled", "pierced", "jabbed")
-	hitsound = 'sound/weapons/bladeslice.ogg'
-	sharpness = IS_SHARP
```
Comment (coiax) [inline code/game/objects/items/weapons/holy_weapons.dm]: Someone defined the pitchfork in the middle of the tribal_knife procs. So I moved it.

**Comment by MrStonedOne** [comment]
So @coiax, unrelated changes in this pr is causing the server to lag for 800 seconds everytime a minigun is deleted.

So good job.

**Comment by coiax** [comment]
Cyberboss made me do it.

**Comment by MrStonedOne** [comment]
@Cyberboss (and @tgstation/commit-access for that matter) no more shoe horning New to Initialize changes into unrelated prs

Some things have to prevent parent behavior, some parents have children that intentionally don't call parent, etc. ie, it needs to be all at once, it needs to be tested, this isn't like removing `:` uses or even converting spawn to addtimer, New to Initialize is a change that has to actually be tested and thought out.

**Comment by coiax** [comment]
Cyberboss didn't actually make me do it.

**Pull request merged**